content/docs/esc/development/_index.md

```diff
@@ -11,3 +11,7 @@ menu:
 ---
 
 The [Pulumi ESC SDK](/docs/esc/development/languages-sdks/) provides a programmatic interface to manage environments, secrets, and configuration directly within your applications. Using the SDK, you can create, update, and delete environments, apply version tags. It also simplifies the secure handling of secrets and configurations, allowing you to retrieve sensitive data like cloud credentials, connection strings, and feature flags at runtime without long-term storage.
+
+[Pulumi Service Provider](/docs/esc/development/psp/) provides ability to create ESC resources like environments, environment permissions, tags and revision tags using a Pulumi program. Setting up configuration and secrets this way, combined with other Pulumi Service resources like Stacks makes setting up a development platform a breeze!
```

Comment:
maybe something like: `provides the ability to create ESC resources like environments, environment permissions, tags, and revision tags using a Pulumi program. Combined with other Pulumi Service resources like Stacks, this makes setting up a development platform a breeze!`
(Also, I remember us planning on preferring the term "version tags" instead of "revision tags" - @dschaller is that still the case?)

Reply:
Hmm, good point on the name, we should stick with one. I did name the PSP resource EnvironmentVersionTag back 3 month ago, so I'll just change it to that